packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -1133,6 +1114,7 @@ export class GraphiQL extends React.Component<GraphiQLProps, GraphiQLState> {
       const leftSize = moveEvent.clientX - getLeft(editorBar) - offset;
       const rightSize = editorBar.clientWidth - leftSize;
       this.setState({ editorFlex: leftSize / rightSize });
+      this._storage.set('editorFlex', JSON.stringify(this.state.editorFlex));
```

Comment:
let's debounce this one, using `utility/debounce`.

Reply:
If I make it like this ` debounce(500, this._storage.set('editorFlex', JSON.stringify(this.state.editorFlex)));`

then this error is showing up
```
Argument of type '{ isQuotaError: boolean; error: any; }' is not assignable to parameter of type '(...args: any[]) => any'.
  Type '{ isQuotaError: boolean; error: any; }' provides no match for the signature '(...args: any[]): any'.ts(2345)
```
Should I make a function for storage set and then pass the variables ?

@acao can you help me with this?

Reply:
@harshithpabbati did not see this before, sorry

Reply:
@harshithpabbati that argument is supposed to be a function it looks like, so I think you'll want to call:

```ts
debounce(500, () => this._storage.set('editorFlex', JSON.stringify(this.state.editorFlex)));
```

Reply:
Okay, cool thanks @acao 

I will update the PR